src/loggers.jl

```diff
@@ -1,112 +1,140 @@
-# Loggers to record properties throughout a simulation

```

Comment:
Why does it diff the unchanged lines? Did the line endings change? Fine as long as they end up unix-style.

Reply:
There were a few places where the line endings were CRLF and I changed them to LF everywhere.